### PR TITLE
Fourth task

### PR DESCRIPTION
**Pull request by synion**

### CODE REVIEW

**Comment by synion** [comment]
@kovson sorry i don't mark you on new pull request

File: spec/features/menage_users_by_admin.rb [hunk 1]
```diff
@@ -33,5 +33,12 @@
     sleep 1
     expect(page).to have_content("example22@o2.pl")
   end
+
+  scenario 'admin can go to admin panel'  do
```
Comment (kovson) [inline spec/features/menage_users_by_admin.rb:37]: It will be nice to also add feature spec which will test that regular users cannot see a link to the admin page.

File: app/assets/javascripts/admin_nav.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+$(document).on('turbolinks:load', function(){
```
Comment (kovson) [inline app/assets/javascripts/admin_nav.js]: Is it necessary?

File: app/assets/stylesheets/admin.scss [hunk 3]
```diff
@@ -0,0 +1,131 @@
+/*
```
Comment (kovson) [inline app/assets/stylesheets/admin.scss]: Same for this.

File: spec/features/menage_users_by_admin.rb (hunk 1 above)
Comment (synion) [reply spec/features/menage_users_by_admin.rb:37]: @kovson can i add this to last regular users feature in the same subject like in last time ??

Comment (kovson) [reply spec/features/menage_users_by_admin.rb:37]: Sure.

File: app/assets/javascripts/admin_nav.js (hunk 2 above)
Comment (synion) [reply app/assets/javascripts/admin_nav.js]: @kovson  i no so sure but when i add this it start working before it`s not

**Comment by kovson** [comment]
@synion 
I like how are you trying to implement this but I would like to make it as simple as possible.

In fact, we can just have a button 'navigate to admin panel' on the top of the page. There is no need to create a dropdown for that. Also, admin page doesn't have to have side admin options coming from the left side of the monitor.

We do not care about UI and UX for now, but you receive a big ➕ for trying to do such thing.

File: app/assets/stylesheets/admin.scss (hunk 3 above)
Comment (synion) [reply app/assets/stylesheets/admin.scss]: This not needed you have right

**Review by synion — COMMENTED** [review]
@kovson 
Thanks for feedback but i don`t know  i should remove this slide navigation and dropdown for repository or it can stay ??

**Comment by kovson** [comment]
@synion 
I think you should remove all fancy features, that are not necessary and are taking yours development time. 
So just a button linking to the admin page, and on admin page some simple navigation. The main goal of this application is to see how are you working with Rails, pull-request system and nit-picking developer who is doing code review.

Bare in mind that your effort and ideas won't be forgotten and it's very nice that you are thinking in such way.

File: app/assets/javascripts/admin.js [hunk 4]
```diff
@@ -0,0 +1,6 @@
+//= require jquery
+//= require bootstrap-sprockets
```
Comment (kovson) [inline app/assets/javascripts/admin.js:2]: Is this file necessary?

File: app/views/layouts/admin_panel.html.haml [hunk 5]
```diff
@@ -0,0 +1,18 @@
+!!!
+%html
+  %head
+    %meta{:name => "viewport", :content => "width=device-width, initial-scale=1.0"}
```
Comment (kovson) [inline app/views/layouts/admin_panel.html.haml:4]: Do we need this?

**Comment by kovson** [comment]
Few quick questions. If we really need those files it can be merged. Nice job and adjustment on the meantime.
